operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/event/EventSourceMetadata.java

```diff
@@ -0,0 +1,9 @@
+package io.javaoperatorsdk.operator.processing.event;
+
+import io.javaoperatorsdk.operator.processing.event.source.EventSource;
+
+public interface EventSourceMetadata {
```

Comment:
Shouldn't we just simply expose `NamedEventSource` ?
This is not really metadata, it is THE event source with name. 
Or rename it to `EventSourceWithMetadata`.

Reply:
I debated only using `NamedEventSource`, don't recall why I didn't actually… 😅

Reply:
Ah yes, didn't want to expose `start` / `stop` but since the `EventSource` is exposed… 🤷🏼 
The alternative would be to expose only what's needed from the `EventSource` via another interface.